packages/next/src/build/webpack-config.ts

```diff
@@ -756,7 +756,7 @@ export default async function getBaseWebpackConfig(
     output: {
       ecma: 5,
       safari10: true,
-      comments: false,
+      comments: config.experimental.minifyLicenses ? false : 'some',
```

Comment:
FYI future readers, `some` is default for `output.comments`
x-ref: https://github.com/terser/terser?tab=readme-ov-file#format-options